includes/admin/customers/class-customer-email-addresses-table.php

```diff
@@ -131,7 +131,14 @@ public function column_email( $item ) {
 
 		// Non-primary email actions
 		if ( 'primary' !== $item_status ) {
```

Comment:
@ashleyfae I believe the issue I had with being able to delete a primary is that we're checking 'status', not 'type'.